### DISCUSSION TITLE
PSD/Project Files for UI Images

### DISCUSSION THREAD

**Discussion by GeoCyberwolf** [Desuuuu/DGUS-reloaded-Klipper#13] [Q&A]
Hi, 

Are these something you have/can provide? I am looking to create a portrait version of the UI, for the Longer LK4 Pro. As this printer uses a DGUS screen, but it comes mounted portrait.

**Comment by Desuuuu** [comment]
I don't really plan on doing that for now.

I've been working on a new touchscreen firmware though, so I'll definitely keep this in mind and try to provide a portrait version with it.
